frontend/src/plugins/impl/DataTablePlugin.tsx

```diff
@@ -191,6 +194,10 @@ export const LoadingDataTableComponent = memo(
     const [sorting, setSorting] = useState<SortingState>([]);
     const [searchQuery, setSearchQuery] = useState<string>("");
     const [filters, setFilters] = useState<ColumnFiltersState>([]);
+    // Column summaries state
+    const [summaries, setSummaries] = useState<ColumnSummaries>({
```

Comment:
i dont think we need to keep track of this state. this is the same as what is inside `useAsyncData`

Reply:
Should be fixed. I remember my first try of simply adding the dependency to `useAsyncData` failed but this time it worked. Not sure why though.